Test/git-issues/git-issue-3804.dfy.expect

```diff
@@ -0,0 +1,7 @@
+git-issue-3804.dfy(9,17): Error: assertion might not hold
+git-issue-3804.dfy(20,17): Error: assertion might not hold
+git-issue-3804.dfy(22,10): Error: assertion might not hold
+git-issue-3804.dfy(31,17): Error: assertion might not hold
+git-issue-3804.dfy(48,10): Error: assertion might not hold
```

Comment:
Oh, I see. The verifier _does_ complain about those lines. Good. Please add comments (like `// error: cannot be verified here`) for those lines, so that these intended failures are evident from just reading the test file.